photo-queue/photo-queue.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: photo-queue
```

Comment:
what about `photo-service-queue`

Reply:
same here this queue does not belong to photo service